morecantile/defaults.py

```diff
@@ -16,7 +16,8 @@
 user_tms_dir = os.environ.get("TILEMATRIXSET_DIRECTORY", None)
 if user_tms_dir:
     tms_paths.extend(list(pathlib.Path(user_tms_dir).glob("*.json")))
-
+    # Sort as users an add a large number of TMS
+    tms_paths.sort()
```

Comment:
I wonder if we could just use `set` instead of list so it's sorted automatically

Reply:
Set would work just as well! I'll update the PR.

Reply:
so weird because I'm getting 

```
{'a', 'b', 'v'}
```

Reply:
let's keep it as a list so it doesn't change the type